README.md

```diff
@@ -120,4 +119,4 @@ Read more about [Bionic Reading](https://bionic-reading.com/about/).
 - Change `version` in `package.json`, eg. `"version": "1.0.0"`
 - Push a new tag to GitHub, eg. `git tag 1.0.0 && git push --tags`
```

Comment:
everything checkouts. appreciate your sharp eye.

mind sharing why these two lines have been taken out please

@X140Yu kindly look over this. i think it is great to have expecially since it was added following #93

Reply:
Yeah, they are still there